tests/phpunit/elementor/core/common/modules/connect/apps/test-common-app.php

```diff
@@ -265,6 +265,25 @@ public function test_http_request__with_connect() {
 		$this->assertEquals( (object) [ 'status' => 'success' ], $result );
 	}
 
+	public function test_get_remote_authorize_url__propagate_query_args() {
+		// Arrange
+		$_REQUEST['black_list'] = 'test';
```

Comment:
?

Reply:
When we pass a value that does not exist in $propagate_query_args_whitelist, the test will return false.

Reply:
Maybe change to `unknown_arg`

Reply:
done